models/staging/edfi_3/stage/stg_ef3__stu_ed_org__cohort_years.sql

```diff
@@ -0,0 +1,19 @@
+with stage_stu_ed_org as (
+    select * from {{ ref('stg_ef3__student_education_organization_associations') }}
+),
+flattened as (
+    select 
+        tenant_code,
+        api_year,
+        k_student,
+        k_student_xyear,
+        ed_org_id,
+        k_lea,
+        k_school,
+        {{ extract_descriptor('value:cohortYearTypeDescriptor::string') }} as cohort_year,
```

Comment:
I'm thinking this should be named `cohort_year_type` to not get confused with the year